index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz App</title>
+    <link rel="stylesheet" href="./styles.css" />
+  </head>
+
+  <body>
+    <header class="header">
+      <h1>Quiz App</h1>
+    </header>
+
+    <!-- QUESTION CARDS -->
+    <section class="card">
+      <div class="centered">
```

Comment:
You don't use the class `centered` in your `style.css` file. Besides, whatever you want to center inside of this `div`, you can safely remove the `div` and pass the CSS rules to the parent `section`.